### ISSUE TITLE
Suggestions by User: mnazmul

### ISSUE THREAD

**Issue by softhash** [softhash/ibilling#2]
CRM
> List contacts
- Need to add Customer Name and company name separately.
- Doesn't show customer picture, need to fix it.
 
Bank & Cash
> Transfer fund from different currency. (Ie: USD >> BDT or USD to SGD or SGD to BDT). U can used xe.com for rate calculation.
> Show balance sheet based on each account & currency
 
 
Sales
> List of the Invoice Table, please add company name
> New Invoice
- Item Code/Part# put 1st column
- Item Name 2nd column
- Invoice to Company and Attn: Person should be separate (See PDF copy top-left)
 
 
Bank & Account:
> New Account add Bank/Branch Name
> Add Account with currency dropdown (Ie: BDT, SGD, USD)
 
 
Product & Service:
> Add Product
- Product Part#
- Add Product SL#
 
My Account:
> Add/Edit Profile:
- Add Picture, Mobile, Facebook, Google+, Linkedin Profile (same like contact more setting).
 
Setting:
> Logo:
- Add Login screen logo (share recommend size, ie: 300x100)
- Add Main Logo (after login top-right)
- Add INVC logo
Basically same logo might be use but different size 

> Manage User:
- Add current Employee role to Finance.
- Add Employee User/role to (without Transaction, Bank & cash, Utility, Setting)
- Add Customer User/role (to see belongs to their order, product, and invoice only)
